runtime/parser/lexer/lexer.go

```diff
@@ -49,6 +49,13 @@ type position struct {
 	column int
 }
 
+type lexerMode uint8
+
+const (
+	normal lexerMode = iota
+	stringInterpolation
```

Comment:
Prefix with the type name, i.e.
```suggestion
	lexerModeNormal lexerMode = iota
	lexerModeStringInterpolation
```